Cargo.toml

```diff
@@ -25,6 +25,7 @@ regex = "1.3"
 linear-map = "1.2"
 serde_base = { version = "^1", optional = true, package = "serde" }
 serde_bytes = { version = "0.11", optional = true }
+serde = { version = "^1", features = ["derive"] }
```

Comment:
Don't add this line -- line 26 is already importing `serde` via `package = "serde"`